breizorro/breizorro.py

```diff
@@ -167,9 +160,18 @@ def main():
     parser.add_argument('--sum-peak', dest='sum_peak', default=None,
                         help='Sum to peak ratio of flux islands to mask in original image.'
                              'e.g. --sum-peak 100 will mask everything with a ratio above 100')
+    parser.add_argument('-ncpu', '--ncpu', dest='ncpu', default=None, type=int,
+                        help='Number of processors to use for cataloging.')
+    parser.add_argument('-beam', '--beam-size', dest='beam', default=None,
+                        help='Average beam size in arcesc incase beam info is missing in image header.')
```

Comment:
I think this breaks with our current CLI convention of only single-letter options for "-". I would use "-j" for ncpu (it's a popular Unix alias) and "-B" for beam.